app/static/js/app.js

```diff
@@ -273,11 +259,11 @@ $( document ).ready(function() {
 });
 
 function hoverInLogo(hoveredOver){
-	document.getElementById("active").id="oldActive";
+	//document.getElementById("active").id="oldActive";
```

Comment:
Ah ok, I removed it because it was throwing null errors. I'll put it back but maybe u can take a look at it.